generators/client/templates/vue/src/main/webapp/app/shared/config/formatter.ts.ejs

```diff
@@ -0,0 +1,122 @@
+function warn(msg: string, err?: Error) {
+    if (typeof console !== 'undefined') {
+        console.warn('[vue-i18n] ' + msg);
+        if (err) {
+            console.warn(err.stack);
+        }
+    }
+}
+
+function isObject(obj: any): boolean {
+    return obj !== null && typeof obj === 'object';
+}
+
+type Token = {
+    type: 'text' | 'named' | 'list' | 'unknown';
+    value: string;
+}
+
+const RE_TOKEN_LIST_VALUE = /^(?:\d)+/;
+const RE_TOKEN_NAMED_VALUE = /^(?:\w)+/;
+
+// as we can't extend the default vue-i18n BaseFormatter we create a own copy of that class with required changes 
+export default class BaseFormatter2 {
+    _caches: { [key: string]: Array<Token> };
+
+    constructor() {
+        this._caches = Object.create(null);
+    }
+
+    interpolate(message: string, values: any): Array<any> {
+        if (!values) {
+            return [message];
+        }
+        // this is custom jhipster logic such that we can reuse the i18n json files for vue-i18n
+        message = message.replace('{{ ', '{').replace(' }}', '}').replace('{{', '{').replace('}}', '}');
```

Comment:
I don't understand the logic here. Shouldn't we use a regexp for doing a replace all ?

Reply:
Good point, will use a regexp in case there are multiple placeholders.